stingray/gti.py

```diff
@@ -968,7 +968,7 @@ def join_equal_gti_boundaries(gti, threshold=0.0):
     return np.asarray(ng)
 
 
-def merge_gtis(gti_list, gti_treatment):
+def merge_gtis(gti_list, strategy):
```

Comment:
Is this change to make the keyword argument consistent everywhere?

Reply:
Yes, also.. I liked "strategy" more than "gti_treatment", it seemed more informative to what that argument does.